src/main/java/com/artipie/api/RestApi.java

```diff
@@ -152,7 +152,12 @@ private void startServices(final RouterBuilder repoRb, final RouterBuilder userR
             new RepoData(this.configsStorage, this.caches.storagesCache()), this.layout
         ).init(repoRb);
         new StorageAliasesRest(this.caches.storagesCache(), asto, this.layout).init(repoRb);
-        Logger.warn(this, "Users API is not available for now");
+        if (this.security.policyStorage().isPresent()) {
```

Comment:
Why we active user management in REST only if security policyStorage is present?

Reply:
Absence of `policyStorage` means that `artipie` credentials format and policy is not used and we do know the format or place where and how users info and policy is stored and thus cannot manage it.